### DISCUSSION TITLE
fabric-resource-loader-v0 causes crash in minecraft 1.20.4

### DISCUSSION THREAD

**Discussion by Menion2k** [FabricMC/fabric-api#3639] [Mod Dev Support]
### Description

Hi all

I am trying to launch via multiMC a farly basic MC 1.20.4 Java Edition. I bring into the mods YetAnotherConfigLib (3.3.2) with Fabric Resource Loader V0 0.11.9 and Fabric APUI 0.96.4 required for Controlify.

But MC crashes at startup (yes I launched once online). The mod that causes the crash is Fabric Resource loader v0 (tried also older version), its pressence is enought to kick the crash.

It complains of:

`java.lang.AbstractMethodError: Receiver class net.fabricmc.fabric.impl.resource.loader.ModResourcePackCreator$$Lambda/0x000001d881b14f60 does not define or inherit an implementation of the resolved method 'abstract net.minecraft.class_3262 method_52424(java.lang.String)' of interface net.minecraft.class_3288$class_7680.`

In another instance I got instead:
`java.lang.NoSuchMethodError: 'net.minecraft.class_2487 net.minecraft.class_2507.method_30613(java.io.File)'`

Attached crashlog and mods directory contents
[crash-2024-03-10_13.53.16-client.txt](https://github.com/FabricMC/fabric/files/14550215/crash-2024-03-10_13.53.16-client.txt)
[mods.zip](https://github.com/FabricMC/fabric/files/14550216/mods.zip)

**Comment by rilakuma123** [comment]
You may want to report this to the author of YACL: https://github.com/isXander/YetAnotherConfigLib/issues

In the meantime, I'd try downgrading YACL to version 3.3.1: https://modrinth.com/mod/yacl/version/3.3.1+1.20.4-fabric

**Reply by Menion2k** [reply]
Hi, tried with same result. Indeed even if I do not enable YACL the problem is still there. It seems frlv0 lack of specializing a pure virtual method fo minecraft

**Comment by Poopooracoocoo** [comment]
I'm not sure if you meant to post in the mod dev category or not, but why do you have the Fabric Resource Loader module in your mods folder? The correct version will be included in Fabric API.

The version that comes with Fabric API 0.96.4+1.20.4:
0.11.19+58f8c0124f

The version you have in your mods folder:
0.11.9+132c48c177

0.11.9+132c48c177 seems to come from Fabric API 0.91.0+1.20.1

**Reply by Menion2k** [reply]
Because YACL requires it, without fabric won't allow MC to start

**Reply by Poopooracoocoo** [reply]
Please try removing it. Fabric API already comes with the right version of the module for your version of Minecraft.
